src/frontend/src/sass/common.scss

```diff
@@ -17,3 +17,4 @@
 @import 'PPQNavBar';
 @import 'PPQResults';
 @import 'PPQConnect';
+@import 'Checkbox';
```

Comment:
Could we move this above page style imports? It is a somewhat minor change now, but css is highly order dependent and if we use this component on a page before this css loads then it may throw an error.